copy/copy.go

```diff
@@ -615,9 +604,24 @@ func (c *copier) copyConfig(ctx context.Context, src types.Image) error {
 		if err != nil {
 			return errors.Wrapf(err, "Error reading config blob %s", srcInfo.Digest)
 		}
-		bar := c.createProgressBar(srcInfo, "config")
-		defer bar.Finish()
-		bar.Start()
+
+		// Create the progress bars
+		progress := mpb.New(mpb.WithWidth(40), mpb.WithOutput(c.progressOutput))
+		// make the name two characters shorted to align with the blob progress bars
+		name := shortDigest(srcInfo.Digest)
+		name = name[0 : len(name)-2]
+		bar := progress.AddBar(srcInfo.Size,
+			mpb.PrependDecorators(
+				decor.Name(fmt.Sprintf("Copying config %s", name)),
+			),
+			mpb.AppendDecorators(
+				decor.CountersKibiByte("%.1f / %.1f"),
+			),
+		)
+		if c.progressOutput == ioutil.Discard {
+			c.Printf("Copying config %s\n", srcInfo.Digest)
+		}
```

Comment:
A quick initial impression: The mechanics of the setup are not entirely trivial, sharing them in a helper (very roughly similar to the `createProgressBar` one) might be more readable.

Reply:
https://github.com/containers/skopeo/pull/602 is green with the updated PR.